Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -60,16 +63,17 @@ open class BasicFormat: SyntaxRewriter {
         !tokenIndentation.isEmpty
       {
         // If the first token in this block is indented, infer the indentation level from it.
-        pushIndentationLevel(increasingIndentationBy: tokenIndentation)
+        let lastNonUserDefinedIndentation = indentationStack.last(where: { !$0.isUserDefined })!.indentation
+        indentationStack.append((indentation: lastNonUserDefinedIndentation + tokenIndentation, isUserDefined: true))
```

Comment:
It's so disconcerting not seeing the same `increase` in both branches. Could `increase` take an indent level and if so add `isUserDefined`?
```
public func increaseIndentationLevel(to indentation: Trivia? = nil) {
  if let indentation = indentation {
    indentationStack.append((indentation, true))
  } else {
    indentationStack.append((currentIndentationLevel + indentationWidth, false))
  }
}
```